jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -361,6 +361,15 @@ def step081_customize_jruby_jar(script_dir, jruby_jar, server_era) {
 }
 
 def step085_customize_hocon_settings(script_dir, settings, server_era) {
+    if ("${MAX_INSTANCES}" != "default") {
```

Comment:
FWIW this commit is refactored enough in the MAX_REQUESTS commit that I think you could just combine them

Reply:
Sounds good, will do.